modules/edu.gemini.seqexec.server/src/main/resources/DhsHdr.xml

```diff
@@ -0,0 +1,1046 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
It was created running a program over the file that defined the channels in the old Seqexec. Then I had to change the types by hand, because the old format doesn't have type information. So yes, the file is machine generated, but not in the sense that it can be generated at build time, if that is what you were thinking.